### ISSUE TITLE
Che workspaces fail on a brand new install 

### ISSUE THREAD

**Issue by triceam** [eclipse-che/che#15806]
I installed Eclipse Che on a brand new Kubernetes 1.16.3_1520 cluster on IBM Cloud by following the installation instructions on https://operatorhub.io/operator/eclipse-che.

Everything seemed to install correctly... I see all the expected pods running, etc... 

When I create a workspace, everything appears OK at first.  For example, you see the loading screen with appropriate messages:

![image](https://user-images.githubusercontent.com/974294/72750331-9506d500-3b8a-11ea-94c3-6d63a8fe62dc.png)

However, once the workspace containers are running, and you expect to see an editor, I just see a "Welcome to nginx!" message:

![image](https://user-images.githubusercontent.com/974294/72750488-fcbd2000-3b8a-11ea-86ac-b7bc700c2a08.png)

I don't see any error messages in any of the pods (operator or workspace), and I'm not quite sure how to debug this further.  I **assume** that something about the Eclipse Che ingress isn't working as expected, but I'm not sure what to check next.   Any help would be greatly appreciated.

Note: before installing via operatorhub instructions, I had to install the IBM Cloud block storage helm chart per https://cloud.ibm.com/docs/containers?topic=containers-block_storage, however that is not the issue b/c I can see storage volumes being created correctly.

(cross posted this issue on the [che-operator repo](https://github.com/eclipse/che-operator/issues/150))

**Comment by tolusha** [comment]
@sleshchenko 
Maybe you have some ideas?

**Comment by triceam** [comment]
Any ideas on this one?  I am still seeing the same thing on a different, Kube 1.15.8_1529 cluster.

**Comment by metlos** [comment]
What is the version of the nginx ingress controller used in your cluster?
```
kubectl get deployment nginx-ingress-controller -n kube-system -o jsonpath='{.spec.template.spec.containers[*].image}'
```
It is strange to me that the page you're seeing is just a homepage of nginx, not some error page. That might suggest that the problem is not with Che itself but rather with nginx controller..

In any way, if your workspace is running then even if you're not seeing the IDE, you should see a couple of ingresses created in the namespace of the workspace. These ingresses make public the services that are serving the containers running in the workspace pod. The error may lie in the interconnections of these objects as well.

**Comment by triceam** [comment]
Turns out that was exactly the problem... The default ingress controller used in new clusters on IBM Cloud is different from the default Nginx ingress controller for other kube clusters, and is not compatible with Eclipse Che.  I could see the Ingress objects in the Kube dashboard had been created, but the rules were never applied.

I reached out the Kube team within IBM and was able to get it working:
1. Disable the default ingress controller (but leave DNS->IP mappings) using `ibmcloud ks alb configure classic --disable-deploymemt` in the `ibmcloud` cli.
2. Install the standard Nginx-based ingress controller using the [instructions here](https://cloud.ibm.com/docs/containers?topic=containers-ingress-user_managed).  Note: I had to make a variation of the [deployment files](https://github.com/kubernetes/ingress-nginx/tree/master/deploy/cloud-generic) b/c as-is, it was giving me RBAC errors.
